src/content/en/updates/2018/11/webcomponents-v0-deprecations.md

```diff
@@ -0,0 +1,102 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A round up of the deprecations and removals in Chrome 71 to help you plan.
+
+{# wf_updated_on: 2018-11-26 #}
+{# wf_published_on: 2018-11-13 #}
+{# wf_tags: deprecations,removals,chrome73,webcomponents,origintrials #}
+{# wf_blink_components: Blink>DOM #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: Web Components v0 deprecation and removal in Chrome 73 to help you plan.#}
+
+{% include "web/updates/_shared/see-all-dep-rem.html" %}
+
+# Web Components v0 deprecation {: .page-title }
+
+{% include "web/_shared/contributors/yoichiosato.html" %}
+
+A part of web interoperability for [Web Components v1](/web/updates/2017/01/webcomponents-org),
+we're deprecating Shadow DOM v0, Custom Element v0 and HTML Imports in Chrome 73.
```

Comment:
I'd throw in links to the h5r articles for reference:
https://www.html5rocks.com/en/tutorials/webcomponents/shadowdom/
https://www.html5rocks.com/en/tutorials/webcomponents/customelements/
https://www.html5rocks.com/en/tutorials/webcomponents/imports/